src/Tribe/Main.php

```diff
@@ -117,6 +117,25 @@ function () {
 			PHP_INT_MAX
 		);
 
+		add_action(
+			'shutdown',
+			function () {
+				/**
+				 * Fires during the shutdown action.
+				 *
+				 * This is mostly useful for testing code. We can trigger this action
+				 * instead of triggering the whole shutdown.
+				 *
+				 * In production code, it can help us only in the sense of adding our own
+				 * actions in a specific order.
+				 *
+				 * @since TBD
+				 */
+				do_action( 'tec_shutdown' );
+			},
+			0
+		);
```

Comment:
Avoid adding new things to `Main.php`. Lets move this to another class that lives in `TEC/Common` namespace, avoid the anonymous fuction here and on the method right above, the one tied to `current_screen`.